hi-ml-histopathology/src/histopathology/datamodules/tcga_crck_module.py

```diff
@@ -31,9 +31,9 @@ def get_splits(self) -> Tuple[TcgaCrck_TilesDataset, TcgaCrck_TilesDataset, Tcga
                                                 group_column=trainval_dataset.SLIDE_ID_COLUMN,
                                                 random_seed=5)
 
-        # if self.number_of_cross_validation_splits > 1:
+        # if self.crossval_count > 1:
         #     # Function get_k_fold_cross_validation_splits() will concatenate train and val splits
-        #     splits = splits.get_k_fold_cross_validation_splits(self.number_of_cross_validation_splits)
+        #     splits = splits.get_k_fold_cross_validation_splits(self.crossval_count)
         # [self.cross_validation_split_index]
```

Comment:
why is this commented?

Reply:
Did not want to touch existing histo code.